firmware/controllers/actuators/electronic_throttle.cpp

```diff
@@ -87,9 +87,9 @@
 #endif
 
 #define ETB_MAX_COUNT 2
-
+	static systime_t start_time;
```

Comment:
don't put these as global statics - move them in to the EtbControl class.  These are per-throttle conditions, and one instance of EtbControl will exist per throttle.